packages/cf-deploy-config-inquirer/src/prompts/app-router-prompts.ts

```diff
@@ -0,0 +1,233 @@
+import type { ListQuestion, ConfirmQuestion, InputQuestion } from '@sap-ux/inquirer-common';
+import { t } from '../i18n';
+import type {
+    CfAppRouterDeployConfigQuestions,
+    CfAppRouterDeployConfigAnswers,
+    CfAppRouterDeployConfigPromptOptions
+} from '../types';
+import { appRouterPromptNames, RouterModuleType } from '../types';
+import { validateMtaPath, validateMtaId, validateAbapService } from './validators';
+import type { Logger } from '@sap-ux/logger';
+import { getCFAbapInstanceChoices, ErrorHandler } from '@sap-ux/inquirer-common';
+
+/**
+ * Generates a prompt for selecting the MTA path.
+ *
+ * @param mtaPath  Mta Path string which allows the user to browse and select a folder for the MTA path
+ * @returns {CfAppRouterDeployConfigQuestions} - The prompt configuration object for selecting the MTA path.
+ */
+function getMtaPathPrompt(mtaPath: string): CfAppRouterDeployConfigQuestions {
+    return {
+        type: 'input',
+        guiOptions: {
+            type: 'folder-browser',
+            breadcrumb: t('prompts.mtaPathBreadcrumbMessage')
+        },
+        name: appRouterPromptNames.mtaPath,
+        message: t('prompts.mtaPathMessage'),
+        default: () => mtaPath,
+        validate: (input: string): string | boolean => validateMtaPath(input)
+    } as InputQuestion<CfAppRouterDeployConfigAnswers>;
+}
+
+/**
+ * Generates a prompt for entering the MTA ID.
+ *
+ * @returns {CfAppRouterDeployConfigQuestions} - The prompt configuration object for entering the MTA ID.
+ */
+function getMtaIdPrompt(): CfAppRouterDeployConfigQuestions {
+    return {
+        type: 'input',
+        guiOptions: {
+            mandatory: true,
+            breadcrumb: true
+        },
+        name: appRouterPromptNames.mtaId,
+        message: t('prompts.mtaIdMessage'),
+        validate: (input: string, previousAnswers: CfAppRouterDeployConfigAnswers): boolean | string =>
+            validateMtaId(input, previousAnswers),
+        filter: (input: string): string => input.replace(/\./g, '-')
+    } as InputQuestion<CfAppRouterDeployConfigAnswers>;
+}
+
+/**
+ * Generates a prompt for entering the MTA description.
+ *
+ * @returns {CfAppRouterDeployConfigQuestions} - The prompt configuration object for entering the MTA description.
+ */
+function getMtaDescriptionPrompt(): CfAppRouterDeployConfigQuestions {
+    return {
+        type: 'input',
+        name: appRouterPromptNames.mtaDescription,
+        guiOptions: {
+            breadcrumb: true
+        },
+        message: t('prompts.mtaDescriptionMessage'),
+        filter: (input: string): string => input?.trim() ?? 'Fiori elements app'
+    } as InputQuestion<CfAppRouterDeployConfigAnswers>;
+}
+
+/**
+ * Generates a prompt for entering the MTA version.
+ *
+ * @returns {CfAppRouterDeployConfigQuestions} - The prompt configuration object for entering the MTA version.
+ */
+function getMtaVersionPrompt(): CfAppRouterDeployConfigQuestions {
```

Comment:
do we require mta version prompt ?

Reply:
Good question! I guess it should be an optional field. Its currently set as a default in tools-suite.

Reply:
I have put it as optional for now - but if its set to default then do we need this prompt then?

Reply:
hmmm...I guess make the prompt optional and in tools-suite we can reconcile the answer. The answer state should return the default value, correct?

Reply:
Yes correct. ok Thanks @longieirl